### ISSUE TITLE
PDF printing creates wrong symbols

### ISSUE THREAD

**Issue by danielzuegner** [cliqz-oss/browser-f#655]
Hi all,

I don't know if this is the right place to post this, but here it is. I am using macOS Sierra and CLIQZ 1.14.1. My system language is German, in case that matters.

I was printing [this pdf](http://www.vldb.org/pvldb/vol8/p581-gatterbauer.pdf), and noticed that some mathematical symbols were mixed up (e.g. '∩' instead of '=') on the print-out version. Now, printing the PDF to disk made it look even weirder (see [attached file](https://github.com/cliqz-oss/browser-f/files/1161605/pdf-symbols.pdf)). I tried some other scientific papers with mathematical symbols and noticed similar results. Printing the PDFs with Safari does not mess up the symbols.

Maybe this is a bug and you may be able to fix it. If it's related to my system, I'd of course be happy to hear ideas of how to solve the problem.

If you need any more information, please let me know.

All the best,

Daniel

**Comment by alver-cliqz** [comment]
Hi, @danielzuegner .

Thanks for the report. This is a right place to do so.

Cliqz browser based on Firefox and using same pdf viewer as Firefox. Can you please check this in latest Firefox version (54.0.1), does this problem reproduced also in it? If yes - we will forward this issue to Mozilla's bugtracker to fix this general problem. If it is only Cliqz issue - we will fix it on our side.

**Comment by danielzuegner** [comment]
Hi @alver-cliqz ,

Thanks for the quick response. I went ahead and installed the latest Firefox version (54.0.1 (64-Bit)) and tried 'printing' to a PDF file (I don't have an actual printer near me right now), and I noticed similar problems with the mathematical symbols (see [uploaded pdf file](https://github.com/cliqz-oss/browser-f/files/1161674/firefox-print.pdf)). I even restarted my computer just to make sure – and printing three times (in Firefox) yielded three different PDFs, with different symbols each time replacing the actual ones. Even the file sizes are significantly different (860KB to 1.2MB). After 'printing' to PDF three times, subsequent prints all yielded the same result and file size. 

If there's anything I can do to help resolve this issue, let me know.

Thanks!

Daniel
